upi/metal/main.tf

```diff
@@ -98,14 +100,16 @@ resource "matchbox_group" "worker" {
 provider "packet" {}
 
 locals {
-  packet_facility = "sjc1"
+  # The packet facility to deploy to - core facilities:
+  # all,ams1,iad2,dc13,dfw2,ewr1,ny5,sin3,sjc1,nrt1
+  packet_facility = "any"
 }
 
 resource "packet_device" "masters" {
   count            = var.master_count
   hostname         = "master-${count.index}.${var.cluster_domain}"
   plan             = "c1.small.x86"
-  facilities       = ["any"]
+  facilities       = [local.packet_facility]
```

Comment:
`local.packet_facility` could be kept as `any` if the `facilities` here (and on line 125) were extracted from the bootstrap node:

```
  facilities = [module.bootstrap.facility] # might need to add facility to the module outputs
```

Reply:
indeed that would be another way to do this. But the current state of the tf can cause the deployment to be distributed across different facilities (e.g. bootstrap in a different facility as the workers and masters)